### PR TITLE
Add missing Creators that were missed when adding team list

### PR DESCRIPTION
**Pull request by VirginiaBalseiro**
These names were missing due to oversight.

EDIT: Because I ended up tidying up the table in my last commit, want to clarify that the commit with the names added is this one: https://github.com/solid/team/pull/53/commits/ce56cd2a37272e364ada9438476deaa6ec28ff90

### CODE REVIEW

File: team.md [hunk 1]
```diff
@@ -32,3 +32,5 @@ Solid Team members must attend Solid Team meetings on a semi-regular basis. They
 | Vincent Tunru     |                                                                                       |                                                    |
 | Michiel de Jong   | [Ponder Source]([https://pondersource.com/)                                         |                                                    |
 | Hadrian Zbarcea   | [Inrupt]([https://www.inrupt.com](https://www.inrupt.com/))                           |                                                    |
+| Giselle Wenban | | https://id.inrupt.com/gisellewenban |
+| Ted Thibodeau | | http://id.myopenlink.net/dataspace/person/tthibodeau#this |
```
Comment (TallTed) [inline team.md:36]:
```suggestion
| Ted Thibodeau     | [OpenLink Software](https://www.openlinksw.com/) | http://id.myopenlink.net/dataspace/person/tthibodeau#this |
```

File: team.md [hunk 2]
```diff
@@ -32,3 +32,5 @@ Solid Team members must attend Solid Team meetings on a semi-regular basis. They
 | Vincent Tunru     |                                                                                       |                                                    |
 | Michiel de Jong   | [Ponder Source]([https://pondersource.com/)                                         |                                                    |
 | Hadrian Zbarcea   | [Inrupt]([https://www.inrupt.com](https://www.inrupt.com/))                           |                                                    |
```
Comment (TallTed) [inline team.md:34]: There are several rows I cannot suggest changes to, that need this same change
```suggestion
| Hadrian Zbarcea   | [Inrupt](https://www.inrupt.com/)                           |                                                    |
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: team.md [hunk 3]
```diff
@@ -31,4 +31,6 @@ Solid Team members must attend Solid Team meetings on a semi-regular basis. They
 | Nicolas Seydoux   | [Inrupt]([https://www.inrupt.com](https://www.inrupt.com/))                           |                                                    |
```
Comment (TallTed) [inline team.md:31]:
```suggestion
| Nicolas Seydoux   | [Inrupt](https://www.inrupt.com/)                           |                                                    |
```

Comment (VirginiaBalseiro) [reply team.md]: I just tidied up the table a bit, which messes up the diff, but I think I covered all these issues.

File: team.md [hunk 4]
```diff
@@ -15,22 +15,22 @@ Solid Team members must attend Solid Team meetings on a semi-regular basis. They
...
+| Sarven Capadisli | | https://csarven.ca/#i |
+| Justin Bingham | [Janeiro Digital](https://www.janeirodigital.com/) | |
+| Kjetil Kjernsmo | | |
+| Nicolas Seydoux |[Inrupt](https://www.inrupt.com/) | |
+| Vincent Tunru | | |
+| Michiel de Jong   | [Ponder Source](https://pondersource.com/) | |
+| Hadrian Zbarcea   | [Inrupt](https://www.inrupt.com/) | |
 | Giselle Wenban | | https://id.inrupt.com/gisellewenban |
-| Ted Thibodeau     | [OpenLink Software](https://www.openlinksw.com/) | http://id.myopenlink.net/dataspace/person/tthibodeau#this |
+| Ted Thibodeau     | [OpenLink Software](https://www.openlinksw.com/) | http:/id.myopenlink.net/dataspace/person/tthibodeau#this |
```
Comment (TallTed) [inline team.md:36]:
```suggestion
| Ted Thibodeau     | [OpenLink Software](https://www.openlinksw.com/) | http://id.myopenlink.net/dataspace/person/tthibodeau#this |
```

**Review by TallTed — APPROVED** [review]
Does still cry out for a number of WebIDs, but they needn't be added in this PR.

**Review by csarven — APPROVED** [review]

File: team.md [hunk 5]
```diff
@@ -15,20 +15,22 @@ Solid Team members must attend Solid Team meetings on a semi-regular basis. They
...
+| Kelly O'Brien | [Inrupt](https://www.inrupt.com/) | |
+| Alain Bourgeois | N/A | |
+| Sarven Capadisli | | https://csarven.ca/#i |
+| Justin Bingham | [Janeiro Digital](https://www.janeirodigital.com/) | |
+| Kjetil Kjernsmo | | |
+| Nicolas Seydoux |[Inrupt](https://www.inrupt.com/) | |
+| Vincent Tunru | | |
+| Michiel de Jong   | [Ponder Source](https://pondersource.com/) | |
+| Hadrian Zbarcea   | [Inrupt](https://www.inrupt.com/) | |
+| Giselle Wenban | | https://id.inrupt.com/gisellewenban |
```
Comment (csarven) [inline team.md:35]:
```suggestion
| Giselle Wenban | Inrupt | https://id.inrupt.com/gisellewenban |
```

**Pull request merged**